avionics/envs/x86/include/stdio_controller.hpp

```diff
@@ -91,28 +139,57 @@ class StdIoController {
      * @return Read value.
      */
     static int requestAnalogRead(uint8_t const pin_id) {
-        run_input_ = false; // input_ will now run at most once more
-        putPacket('a', &pin_id,
-                  1);  // ensures that at least one more SIM packet exists
-        input_.join(); // so this won't block
+        uint8_t packet_id = static_cast<uint8_t>(PacketIds::analog_read);
```

Comment:
I wonder if constexpr will work here, just to indicate that this variable isn't going to change (and/or ALL_CAPS to indicate constant)